news/models.py

```diff
@@ -18,13 +18,25 @@
 from wagtail.snippets.edit_handlers import SnippetChooserPanel
 from wagtail.snippets.models import register_snippet
 from wagtail.api import APIField
+from wagtail.images.api.fields import ImageRenditionField
 
 from modelcluster.fields import ParentalKey
 from modelcluster.contrib.taggit import ClusterTaggableManager
 from taggit.models import TaggedItemBase
 from openstax.functions import build_image_url
 from snippets.models import NewsSource
 
+class ImageChooserBlock(ImageChooserBlock):
+    def get_api_representation(self, value, context=None):
+        if value:
+            print(dir(value))
+            return {
+                'id': value.id,
+                'title': value.title,
+                'original': value.get_rendition('original').attrs_dict,
+                'small': value.get_rendition('width-300').attrs_dict,
```

Comment:
Is it straightforward to change to width-420? Or are there some predefined available values you have to work with?